bitsandbytes/cextension.py

```diff
@@ -67,6 +63,9 @@ def __init__(self, lib: ct.CDLL):
     def __getattr__(self, item):
         return getattr(self._lib, item)
 
+    def __getitem__(self, item):
+        return getattr(self._lib, item)
```

Comment:
Why is this required? I don't think the library should look mapping-like.

`getattr(lib, "foo")` would work.